spec/migrations/20200912032913_add_ancestry_to_resource_pool_spec.rb

```diff
@@ -0,0 +1,181 @@
+require_migration
+
+describe AddAncestryToResourcePool do
+  let(:rel_stub) { migration_stub(:Relationship) }
+  let(:resource_pool_stub) { migration_stub :ResourcePool }
+  let(:resource_pool) { resource_pool_stub.create! }
+  let(:resource_pool2) { resource_pool_stub.create! }
+  let(:resource_pool3) { resource_pool_stub.create! }
+  let(:resource_pool4) { resource_pool_stub.create! }
+  let(:resource_pool5) { resource_pool_stub.create! }
+  let(:resource_pool6) { resource_pool_stub.create! }
+  let(:resource_pool7) { resource_pool_stub.create! }
+  let(:resource_pool8) { resource_pool_stub.create! }
+  let(:root) { resource_pool_stub.create! }
+
+  migration_context :up do
+    context "single parent/child rel" do
+      it 'updates ancestry' do
+        parent_rel = rel_stub.create!(:relationship => 'genealogy', :ancestry => nil, :resource_type => 'ResourcePool', :resource_id => root.id)
+        rel_stub.create!(:relationship => 'genealogy', :ancestry => parent_rel.id, :resource_type => 'ResourcePool', :resource_id => resource_pool.id)
```

Comment:
not sure if this is a good idea:

does it make sense to add `genealogy` or maybe resource_pool method in the model so we don't have all this gook?